src/core/lint-hrefs-ids.js

```diff
@@ -0,0 +1,21 @@
+// Module core/lint-hrefs-ids
+// Warns about href's that do not have corresponding id's
+
+import { pub } from "core/pubsubhub";
+
+export const name = "core/lint-hrefs-ids";
+
+export function run(conf, doc, cb) {
```

Comment:
I see that linting is skipped if `specStatus` is `"unofficial"` [here](https://github.com/w3c/respec/blob/develop/src/w3c/linter.js#L46). But I'm working on documents totally unrelated to W3C with help of respec. Should this linting really work only for "official" docs? Can we make it also cover my "unofficial" use-case somehow?